docs/userguides/clis.md

```diff
@@ -105,17 +105,21 @@ Additionally, specify `ecosystem`, `network`, or `provider` in your command func
 import click
 from ape.cli import ConnectedProviderCommand
 
-@click.command(cls=ConnectedProviderCommand)
```

Comment:
this example wasn't demonstrating Ape's scripting system but was rather showing how you can use our CLI tools (anywhere).